packages/browser-destinations/destinations/cdpresolution/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "@segment/analytics-browser-actions-cdpresolution",
+  "version": "1.0.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/segmentio/action-destinations",
+    "directory": "packages/browser-destinations/destinations/cdpresolution"
+  },
+  "main": "./dist/cjs",
+  "module": "./dist/esm",
+  "scripts": {
+    "build": "yarn build:esm && yarn build:cjs",
+    "build:cjs": "tsc --module commonjs --outDir ./dist/cjs",
+    "build:esm": "tsc --outDir ./dist/esm"
+  },
+  "typings": "./dist/esm",
+  "dependencies": {
+    "@segment/browser-destination-runtime": "^1.4.0"
```

```suggestion
    "@segment/browser-destination-runtime": "^1.12.2"
```

Hi @ajaysikri , could you update the version as specified please?

---

Hi Joe,

I have updated the version as requested.

Thanks, Ajay